### PR TITLE
Don't disclose emails of all users when sending out emails

### LINKED ISSUE #4417: [Privacy] Gitea leaks hidden email addresses
- Gitea version (or commit ref): 3e445cc
- Git version: not relevant
- Operating system: not relevant
- Database (use `[x]`):
  - [ ] PostgreSQL
  - [ ] MySQL
  - [ ] MSSQL
  - [ ] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [x] Yes (provide example URL)
  - [ ] No
  - [ ] Not relevant
- Log gist:

### PR DESCRIPTION
**Pull request by techknowlogick**
Fix #4417

### CODE REVIEW

File: models/issue_mail.go [hunk 1]
```diff
@@ -87,7 +88,9 @@ func mailIssueCommentToParticipants(e Engine, issue *Issue, doer *User, content
 		names = append(names, participants[i].Name)
 	}
 
-	SendIssueCommentMail(issue, doer, content, comment, tos)
+	for _, to := range tos {
+		SendIssueCommentMail(issue, doer, content, comment, []string{to})
```
Comment (jonasfranz) [inline models/issue_mail.go:92]: Emails should be send to all users which have there email adresse disclosed at once, shouldn't them?

Comment (cezar97) [reply models/issue_mail.go]: That's what I suggested in the related issue. But I dont't know if it's possilble to do.

Comment (techknowlogick) [reply models/issue_mail.go:92]: yes, that is possible, however I wanted to resolve the security issue before adding any enhancements.

**Review by sapk — APPROVED** [review]

**Comment by lunny** [comment]
LGTM

**Comment by techknowlogick** [comment]
Thanks @lunny. Backport PR created.

**Pull request merged**